Modules/Options/QuestieOptionsDefaults.lua

```diff
@@ -68,25 +68,27 @@ function QuestieOptionsDefaults:Load()
             autoTrackQuests = true,
             trackerShowCompleteQuests = true,
             collapseCompletedQuests = false,
+            hideCompletedObjectives = false,
```

Comment:
This should be added to a new entry in the `Migration.lua` so existing configurations pick up this new value.